lms/static/js/groups/views/cohort_editor.js

```diff
@@ -18,11 +27,38 @@
         render: function() {
             this.$el.html(this.template({
                 cohort: this.model,
-                advanced_settings_url: this.advanced_settings_url
+                contentGroups: this.contentGroups,
+                studioAdvancedSettingsUrl: this.context.studioAdvancedSettingsUrl
             }));
+            this.cohortFormView = new CohortFormView({
+                model: this.model,
+                contentGroups: this.contentGroups,
+                context: this.context
+            });
+            this.cohortFormView.render();
+            this.$('.tab-content-settings').append(this.cohortFormView.$el);
             return this;
         },
 
+        selectTab: function(event) {
+            var tabElement = $(event.currentTarget),
+                tabName = tabElement.data('tab');
+            event.preventDefault();
+            this.$('.wrapper-tabs .tab').removeClass('is-selected');
```

Comment:
In some of the Solutions PRs, I've seen Aria roles being update as well. We should have a task for @cptvitamin to do an accessibility review. It might be good for him to take a look at your sandbox now.